components/AppBar/src/MDCAppBarViewController.m

```diff
@@ -276,6 +288,15 @@ - (void)dismissSelf {
   }
 }
 
+#pragma mark - Private
+
+- (void)adjustHeightBasedOnHeaderStackView {
+  CGFloat heightSum = 0;
+  heightSum += [self.headerStackView.topBar sizeThatFits:self.view.frame.size].height;
+  heightSum += [self.headerStackView.bottomBar sizeThatFits:self.view.frame.size].height;
+  self.headerView.minimumHeight = self.headerView.maximumHeight = heightSum;
```

Comment:
Readability:

```objc
self.headerView.minimumHeight = heightSum;
self.headerView.maximumHeight = heightSum;
```

Reply:
Addressed in 1b97d553ff0a58d5597bbe7382a338085125677f.